client/state/plans/assembler.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+
+export const createPlanObject = plan => {
+	return {
+		androidSku: String( plan.android_sku ),
+		appleSku: String( plan.apple_sku ),
+		available: String( plan.available ),
+		billPeriod: Number( plan.bill_period ),
+		billPeriodLabel: String( plan.bill_period_label ),
+		cost: Number( plan.cost ),
+		capability: String( plan.capability ),
```

Comment:
Could we maybe resolve the assembler vs no assembler pattern? I personally would prefer having useable and consistent json endpoints, vs having to do client side conversions on each prop. We already enforce what types we expect in our schema, so this will be extra overhead.